dvc/repo/experiments/queue/base.py

```diff
@@ -170,6 +170,14 @@ def get_result(self, entry: QueueEntry) -> Optional[ExecutorResult]:
         This method blocks until the specified item has been collected.
         """
 
+    @abstractmethod
+    def kill(self, name: str) -> None:
```

Comment:
Can we clarify what `name` is expected to be here? I think we probably need it to be a `revs` argument (which can be either a `-n` name or a stash revision) similar to how `BaseStashQueue.remove()` works.

In the `LocalCeleryQueue` implementation, we will need to iterate over the active `QueueEntry`'s to find the entry that matches the specified stash rev or name, and then kill the appropriate process (which is named according to `entry.stash_rev` in the `setup_exp` task).